interfaces/prompting/requestprompts/requestprompts.go

```diff
@@ -17,6 +17,8 @@
  *
  */
 
+// Package requestrules provides the manager for request prompting related to
```

Comment:
s/the manager .../support for holding not-yet replied request prompts for AppArmor prompting./